guide/C/ch_capgain.xml

```diff
@@ -38,7 +38,7 @@
       taxation.
     </para>
 
-    <para>Consumable and disposable items (eg: food, gas for your car, or printer paper) are obviously not
+    <para>Consumable and disposable items (for example, food, gas for your car, or printer paper) are obviously not
       involved. Thus, even though the new clothes you recently bought will certainly depreciate, you
       would not want to track this depreciation since you have no intention of reselling the clothes
       and there is no tax implications to the depreciation on clothing. So, for this example, the
```

Comment:
@fellen it doesn't matter whether the tax folks think it's taxable, nor whether you thought you were holding the item for investment. Money has to come from somewhere and go to somewhere. If you buy an asset--no matter what kind--and hold it on your books instead of expensing it, then sell it later for a different amount than you paid, there's a gain or loss that has to be accounted for. That is usually called a capital gain or loss in the US and the general category is capital gains. If that bugs you we could call the chapter something like "Gains and Losses on Assets".

Reply:
Let's take `Gains and Losses on Assets`